postgres/immich-docker-entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Also does the container even have bash? I thought it only had sh